website/src/app/core/services/data-distributions/data-distributions.service.ts

```diff
@@ -2,13 +2,19 @@ import { EMPTY_TABLE_DATA_DIRECTORY, TableDataDirectory } from './../../models/t
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { EMPTY_TABLE_DATA, TableData } from '../../models/table-data.model';
+import { Dataset } from '../../models/dataset.model';
+
+const SUB_LABEL_FLAG = 'Tox lab flag';
+const ASSETS_DIRECTORY = 'assets/generated/aggregate-table-data.json';
```

Comment:
This is not an assets directory... Consider renaming to something like DISTRIBUTIONS_CONFIG_PATH